docs/production/settings.md

```diff
@@ -82,6 +82,58 @@ support for included HTML).  A good approach is to use paths like
 `/etc/zulip/terms.md`, so that it's easy to back up your policy
 configuration along with your other Zulip server configuration.
 
+```eval_rst
+.. _email-gateway:
+```
```

Comment:
What do you need this `eval_rst` here for -- I guess it's to make `#email-gateway` work in addition to `#the-email-gateway`?